usefulstuff.c

```diff
@@ -650,6 +650,28 @@ int xdebug_format_output_filename(char **filename, char *format, char *script_na
 					}
 				}	break;
 
+				case 'K': { /* XDEBUG_FILE in cookie */
```

Comment:
Why did you pick "K" ? Can we make it "X", according to the xdebug_eXtra_info ?

Reply:
I'm new  to this